docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,154 @@
+
+To delete a snapshot request, simply exchange `submitrequest` with `cancelrequest`. For example:
+
+```
+peer snapshot cancelrequest -C testchannel -b 1000 --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+If you submit the `listpending` command again, the snapshot should no longer appear.
+
+### Contents of a snapshot
+
+Snapshots are stored on a peer in a snapshot folder that is further organized by channel name and the block number when the snapshot was taken: `{snapshotsFolder}/completed/{channelName}/{lastBlockNumberInSnapshot}`. Once the peer generates a snapshot in this folder, the peer does not use that folder for any purpose and it is safe to transfer or compress the snapshot using external tools.
+
+As mentioned above, the snapshot contains four folders:
+
+* **Public state**
+  * This includes the latest state of all of the assets on the channel (for example, the current owner of a car).
+* **Private data hashes**
+  * Recall from our documentation on private data that while the actual transaction data of a private data transaction is not stored on the public ledger of the channel, hashes of the data are stored. This allows the private data to be verified against the hash, for example by an organization that is added to a private data collection.
+* **Transactions IDs**
+  * This consists of all of the transaction IDs from the beginning of the ledger to the last block in the snapshot. Starting with Fabric v2.0, entries in the block store index take the form of `TxID_BlkNum_TxNum`. This index can be filtered for the transactions IDs that are added after the snapshot height by evaluating block number (`BlkNum`) portion of the index entry.
+* **Collection config history**
+  * This contains the private data collections the peer is a member of.
+
+In addition, the snapshot contains two metadata files that help the data in the snapshot to be verified. This snapshot metadata file is expected to be same across snapshots of a channel for a particular height.
+
+One of these files contains a JSON record called `_snapshot_signable_metadata.json` with the following fields:
+
+* `channel_name`: the name of the channel.
+* `last_block_number`: the block height when the snapshot was taken.
+* `last_block_hash`: a hash of the block at the height the snapshot was taken.
+* `previous_block_hash`: a hash of the block prior to the `last_block`.
+* `state_db_type` (the value of this field will be either CouchDB or SimpleKeyValueDB (also known as LevelDB).
+* `snapshot_files_raw_hashes`, is a JSON record that contains the hashes of the files present in `_snapshot_additional_metadata.json`.
+
```

Comment:
>hashes of the files present in `_snapshot_additional_metadata.json`.

hashes of the files present for the above mentioned data items - i.e., `Public state`, `Private data hashes`, `Transactions IDs`, and `Collection config history`

Reply:
I think that I did not make myself clear enough in this. "hashes of the files present in _snapshot_additional_metadata.json" is a wrong/meaningless statement. So, I suggested a replacement.

Reply:
Ok I think I understand now. Changing the relevant bullet to: 

`snapshot_files_raw_hashes`, is a JSON record that contains the hashes of the files above